README.md

```diff
@@ -10,11 +10,13 @@ This repository contains the implementation of these experiments built using [AI
 
 ## 🤖️ Implemented simulations:
 
+Before attempting to reproduce these simulations, ensure that you have completed [basic setup](#💻-installation).
+
 
 ### Gravity Bias
 This experiment tests an agent's ability to recognize the role gravity plays in objects' movements. A number of rewards is dropped through an opaque tube into one of three receptables, and the agent has to infer in which receptable the reward lands.
 
-Unlike the previous experiment, this experiment was built using Unity3D instead of AI2-THOR. To set up the experiment, you need to:
+Unlike some other experiments, this experiment was built using Unity3D instead of AI2-THOR. To set up the experiment, you need to:
```

Comment:
clarified wording here because there are no previous experiments, as this is the first listed in the README